pennylane/_qutrit_device.py

```diff
@@ -305,9 +305,12 @@ def marginal_prob(self, prob, wires=None):
         prob = self._reshape(prob, [3] * self.num_wires)
 
         # sum over all inactive wires
-        prob = self._flatten(self._reduce_sum(prob, inactive_device_wires.labels))
-        # TODO: Add case for when inactive_device_wires is not an instance of Wires once
-        # default qutrit device is added
+        # hotfix to catch when default.qutrit uses this method
```

Comment:
Just noting here for other reviewers that this is done in `QubitDevice` in exactly the same way.